document/FileFormat.mdk

```diff
@@ -2696,31 +2703,318 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+|                  | name of the variable        | declaration has an          | declaration has an          |
+|                  | being declared. If          | initializer, in which       | initializer, in which case, |
+|                  | multiple variables are      | case, the value of          | the names and values of     |
+|                  | declared in the same        | the initializer             | any variables participating |
+|                  | statement, a separate       | expression.                 | in the initializer          |
+|                  | `annotatedCodeLocation`     | Absent: If the              | expression.                 |
+|                  | object shall be created     | declaration has no          | Absent: If the              |
+|                  | for each declaration that   | initializer.                | declaration has no          |
+|                  | the tool creating the       |                             | initializer.                |
+|                  | log wishes to represent.    |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `functionEnter`  | Recommended: The            | Absent                      | Absent                      |
+|                  | fully qualified name of     |                             |                             |
+|                  | the function being          |                             |                             |
+|                  | entered.                    |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `functionExit`   | Recommended: The            | Optional: If the function   | Absent                      |
+|                  | fully qualified name of     | returns a value or          |                             |
+|                  | the function being          | values, in which case,      |                             |
+|                  | returned from.              | string representations      |                             |
+|                  |                             | of the value or values      |                             |
+|                  |                             | returned.                   |                             |
+|                  |                             | Absent: If the function     |                             |
+|                  |                             | does not return a value.    |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `functionReturn` | Recommended: The            | Optional: If the function   | Optional: If the function   |
```

Comment:
>Recommended [](start = 21, length = 11)

I think this would only be recommended if the functionReturn is not nested within a functionEnter/functionExit pair, do you agree?